quafu/tasks/tasks.py

```diff
@@ -253,9 +253,7 @@ def send(self,
             raise UserError()
         else:
             res_dict = response.json()
-            import pprint
```

Comment:
I assume that you want to use the verbose flag to control whether we print response, so you finally decide to remove this now? If so, maybe the verbose flag should be deleted.

Reply:
Actually I intended to help debug by setting ``verbose=True``, yet it was not completed. Let's delete it for now. By the way I just noticed status_code in two level in master branch were not seperated yet?

Reply:
I believe this is a mistake I made when I merge stable/0.3 branch into master. When resolving conflicts, I mistakenly removed the change in stable/0.3 branch and kept the master branch change. I will fix this later.